src/components/calcite-value-list/calcite-value-list.stories.ts

```diff
@@ -105,6 +105,23 @@ export const basic = (): string =>
     `
   );
 
+export const withContentSlot = (): string =>
+  create(
+    "calcite-value-list",
+    createAttributes(),
+    html`
+      <calcite-value-list-item label="Default label and description" description="used as content" value="dogs">
+        ${action}
+      </calcite-value-list-item>
+      <calcite-value-list-item value="cats">
+        <div slot="content">
+          <calcite-input placeholder="slot content"></calcite-input>
```

Comment:
I think this will be problematic, the role of the list-item is a checkbox. The checkbox can't contain an input inside of it I don't think. The DOM structure and roles would need to be reconfigured.

Can we make sure that this HTML passes aXe?